### PR TITLE
Fcm tutorial

### LINKED ISSUE #67: Document how to setup FCM legacy API
On the firebase console.

Split from https://github.com/UnifiedPush/common-proxies/issues/47

Maybe a link to another source is enough ? eg. https://documentation.onesignal.com/docs/generate-a-google-server-api-key

### PR DESCRIPTION
**Pull request by karmanyaahm**
I'm adding Git LFS here, so these images don't stay part of this repository's (hopefully long long) future forever.
We can also add more large files, images, etc this way without impacting repo performance.

Edit: closes #67 
Edit: also, kinda cool how we've gotten to so many issues and PRs in total on this repo

### CODE REVIEW

**Comment by karmanyaahm** [comment]
This is what it looks like right now. 
Screenshot 2022-07-21 at 19-56-21 Embedded FCM Distributor

**Comment by karmanyaahm** [comment]
@jld3103 any comments?

**Comment by karmanyaahm** [comment]
I also need to link to this from common proxies once it's done

**Comment by jld3103** [comment]
LGTM

**Comment by karmanyaahm** [comment]
kk, @p1gp1g feel free to merge if/when you think this is good

File: content/developers/embedded_fcm.md [hunk 1]
```diff
@@ -112,3 +112,45 @@ location  /FCM {
...
+If you already have an Android app added to your project, skip this section.
+
+1. Click on the Android logo under 'Your apps' ![See adjacent text](/big_img/fcm_setup/gen_settings_no_app.png)
+1. Enter your application ID into the app creation menu. This must be the same as what you provide to the Android app's configuration. ![See adjacent text](/big_img/fcm_setup/create_app_1.png)
+1. Press 'Next', 'Next', 'Continue' and go back to your settings console.
+
+### Setting up your app on Android (FOSS FCM lib)
+
+- From under 'Your project', copy the 'Project number', it looks something like the number '123456789012' (12 digits)
+- Put this in your [EmbeddedDistributor.googleProjectNumber](#android)
```
Comment (p1gp1g) [inline content/developers/embedded_fcm.md:149]:
```suggestion
- Under the menu *Your project*, copy the *Project number*. It is a 12 digits number, for instance `123456789012`.
- Use this value for the [EmbeddedDistributor.googleProjectNumber](#android)
```

File: content/developers/embedded_fcm.md [hunk 2]
```diff
@@ -112,3 +112,45 @@ location  /FCM {
...
+
+### Setting up your app on Android (Google FCM lib)
+
+- From under 'Your apps', download 'google-services.json'
```
Comment (p1gp1g) [inline content/developers/embedded_fcm.md:153]:
```suggestion
- Under the menu *Your apps*, download *google-services.json*.
```

**Comment by karmanyaahm** [comment]
I added all your fixes (and italicized some more menus) and squashed the commits

**Pull request merged**